test/extended/fixtures/forcepull-false-s2i-is.json

```diff
@@ -0,0 +1,14 @@
+{
+    
+    "kind": "ImageStream",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "s2i-forcepull-false-ruby-20-centos7",
+        "creationTimestamp": null
+    },
+    "spec": {},
+    "status": {
+        "dockerImageRepository": ""
+    }
+    
+}
```

Comment:
you should be able to combine the buildconfig and imagestream definitions into a single json file, you can have lists of api objects (that's basically what processing a template produces).  I think it would simplify this by reducing the number of files, and the amount of calls your test code has to make to set things up.